committeeoversightapp/templates/partials/compare_committees_over_congresses_table.html

```diff
@@ -0,0 +1,45 @@
+<div class="card mt-5 col-12 col-lg-10 mx-auto card-shadow">
+  <h2 class="text-center mt-4">{{title}}</h2>
+  <div class="card-body">
+    <div class="table-responsive">
+      <table class="committee-table table table-borderless table-striped table-sm text-center"
+      >
+      <thead>
+        <tr class="landing-row">
+          <th class="align-top">Committee</th>
```

Comment:
It seems like some of the older Congresses consistently don't have sort icons, do you know why that might be?
<img width="1131" alt="Screen Shot 2019-11-12 at 1 26 28 PM" src="https://user-images.githubusercontent.com/14170650/68703360-18cf9000-0550-11ea-8242-7d853d69b6df.png">

Reply:
Good catch! It's because I forgot to delete this line from the `DataTables` function on this page:

```
          "columnDefs": [
            { "orderable": false, "targets": [3, 4, 5] },
          ],
```